src/index.ts

```diff
@@ -49,11 +50,16 @@ export class InteractionManager {
   constructor(
     renderer: THREE.Renderer,
     camera: THREE.Camera,
-    domElement: HTMLElement
+    domElement: HTMLElement,
+    dontBindEventsOnBody: boolean | undefined
```

Comment:
Minor nitpick: `dontBindEventsOnBody?: boolean` would be the typical way of writing this.
Other than that, nicely made! 👍